### ISSUE TITLE
Plugin fails to convert to kotlin

### ISSUE THREAD

**Issue by Martmists-GH** [valich/jflexToKotlin#1]
Following instructions from Jetbrains/Markdown, and after manually updating the plugin to build for latest stable IntelliJ, Convert JFlex to Kotlin does nothing.

I tried using Convert Java to Kotlin as well, but that seemed to stop early on. The .flex file used is sfm.flex from the Jetbrains/Markdown repo.

**Comment by Martmists-GH** [comment]
Update: This seemed to be an issue with the new java-to-kotlin converter in IntelliJ. After using the old version, I get the following error running Convert JFLEX to Kotlin:

```kotlin
com.intellij.util.IncorrectOperationException: Incorrect expression '"\u0001\u0000\u0001\u0001\u0001\u0002\u0001\u0003\u0001\u0004\u0001\u0005\u0001\u0006\u0001\u0007\u0001\u0008\u0001\u0009\u0001\n\u0001\u000b\u0001\u000c\u0006\r\u0001\u000e\u0001\u000f\u0001\u0010\u0003\r\u0001\u0011\u0001\r\u0001\u0012\u0001\u0013\u0008\r\u0001\u0014\u0001\u0015\u0001\u0016\u0001\r\u0001\u0017\u0001\u0018\u0001\u0019\u0009\r\u0001\u001a\u0008\n\u0001\u001b\u0001\u001c\u0001\u001d\u0001\u001e\u0001\u001f\u0001\u0020\u0001\u0021\u0001\u0022\u0001\u0023\u0001\u0024\u0001\u0025\u0001\u0026\u0002\n\u0001\r\u0001\u0027\u0003\n\u0001\u0028\u0008\n\u0001\u0029\u0001\u002a\u0005\r\u0001\u002b\u0001\u002c\u0009\n\u0001\u002d\u0002\n\u0001\u002e\u0004\n\u0001\u002f\u0001\u0030\u0004\n\u0001\u0031\u0001\u0032\u0001\u0033\u0003\n\u0029\r\u0001\u0034\u0003\r\u0001\u0035\u0001\u0036\u0004\r\u0001\u0037\n\n\u0001\u0038\u0381\n"'
	at com.intellij.psi.impl.PsiJavaParserFacadeImpl.newException(PsiJavaParserFacadeImpl.java:388)
	at com.intellij.psi.impl.PsiJavaParserFacadeImpl.createExpressionFromText(PsiJavaParserFacadeImpl.java:295)
	at com.intellij.psi.impl.PsiElementFactoryImpl.createExpressionFromText(PsiElementFactoryImpl.java:642)
	at com.github.valich.jflextokotlin.actions.ConvertLexerToKotlin.createShorterStringConcat(ConvertLexerToKotlin.kt:131)
	at com.github.valich.jflextokotlin.actions.ConvertLexerToKotlin.access$createShorterStringConcat(ConvertLexerToKotlin.kt:18)
	at com.github.valich.jflextokotlin.actions.ConvertLexerToKotlin$convertStringLiterals$1$1.visitPolyadicExpression(ConvertLexerToKotlin.kt:49)
	at com.intellij.psi.impl.source.tree.java.PsiPolyadicExpressionImpl.accept(PsiPolyadicExpressionImpl.java:112)
	at com.intellij.psi.impl.source.tree.CompositePsiElement.acceptChildren(CompositePsiElement.java:65)
	at com.intellij.psi.JavaRecursiveElementVisitor.visitElement(JavaRecursiveElementVisitor.java:44)
	at com.intellij.psi.JavaElementVisitor.visitExpressionList(JavaElementVisitor.java:96)
	at com.intellij.psi.impl.source.tree.java.PsiExpressionListImpl.accept(PsiExpressionListImpl.java:159)
	at com.intellij.psi.impl.source.tree.CompositePsiElement.acceptChildren(CompositePsiElement.java:65)
	at com.intellij.psi.JavaRecursiveElementVisitor.visitElement(JavaRecursiveElementVisitor.java:44)
	at com.intellij.psi.JavaElementVisitor.visitExpression(JavaElementVisitor.java:92)
	at com.intellij.psi.JavaElementVisitor.visitCallExpression(JavaElementVisitor.java:176)
	at com.intellij.psi.JavaElementVisitor.visitMethodCallExpression(JavaElementVisitor.java:172)
	at com.intellij.psi.impl.source.tree.java.PsiMethodCallExpressionImpl.accept(PsiMethodCallExpressionImpl.java:126)
	at com.intellij.psi.impl.source.tree.SharedImplUtil.acceptChildren(SharedImplUtil.java:185)
	at com.intellij.psi.impl.source.JavaStubPsiElement.acceptChildren(JavaStubPsiElement.java:113)
	at com.intellij.psi.JavaRecursiveElementVisitor.visitElement(JavaRecursiveElementVisitor.java:44)
	at com.intellij.psi.JavaElementVisitor.visitVariable(JavaElementVisitor.java:332)
	at com.intellij.psi.JavaElementVisitor.visitField(JavaElementVisitor.java:108)
	at com.intellij.psi.impl.source.PsiFieldImpl.accept(PsiFieldImpl.java:363)
	at com.intellij.psi.impl.source.tree.SharedImplUtil.acceptChildren(SharedImplUtil.java:185)
	at com.intellij.psi.impl.source.JavaStubPsiElement.acceptChildren(JavaStubPsiElement.java:113)
	at com.intellij.psi.JavaRecursiveElementVisitor.visitElement(JavaRecursiveElementVisitor.java:44)
	at com.intellij.psi.JavaElementVisitor.visitClass(JavaElementVisitor.java:44)
	at com.intellij.psi.impl.source.PsiClassImpl.accept(PsiClassImpl.java:464)
	at com.intellij.psi.impl.source.tree.SharedImplUtil.acceptChildren(SharedImplUtil.java:185)
	at com.intellij.psi.impl.source.PsiFileImpl.acceptChildren(PsiFileImpl.java:754)
	at com.intellij.psi.JavaRecursiveElementVisitor.visitElement(JavaRecursiveElementVisitor.java:44)
	at com.intellij.psi.PsiElementVisitor.visitFile(PsiElementVisitor.java:35)
	at com.intellij.psi.JavaElementVisitor.visitJavaFile(JavaElementVisitor.java:340)
	at com.intellij.psi.impl.source.PsiJavaFileBaseImpl.accept(PsiJavaFileBaseImpl.java:480)
	at com.github.valich.jflextokotlin.actions.ConvertLexerToKotlin$convertStringLiterals$1.invoke(ConvertLexerToKotlin.kt:43)
	at com.github.valich.jflextokotlin.actions.ConvertLexerToKotlin$convertStringLiterals$1.invoke(ConvertLexerToKotlin.kt:42)
	at org.jetbrains.kotlin.idea.util.application.ApplicationUtilsKt$sam$com_intellij_openapi_util_Computable$0.compute(ApplicationUtils.kt)
	at com.intellij.openapi.application.impl.ApplicationImpl.lambda$runWriteAction$13(ApplicationImpl.java:1032)
	at com.intellij.openapi.application.impl.ApplicationImpl.runWriteActionWithClass(ApplicationImpl.java:1011)
	at com.intellij.openapi.application.impl.ApplicationImpl.runWriteAction(ApplicationImpl.java:1032)
	at org.jetbrains.kotlin.idea.util.application.ApplicationUtilsKt.runWriteAction(ApplicationUtils.kt:20)
	at org.jetbrains.kotlin.idea.util.application.ApplicationUtilsKt$executeWriteCommand$1.run(ApplicationUtils.kt:38)
	at com.intellij.openapi.command.impl.CoreCommandProcessor.executeCommand(CoreCommandProcessor.java:219)
	at com.intellij.openapi.command.impl.CoreCommandProcessor.executeCommand(CoreCommandProcessor.java:174)
	at com.intellij.openapi.command.impl.CoreCommandProcessor.executeCommand(CoreCommandProcessor.java:164)
	at com.intellij.openapi.command.impl.CoreCommandProcessor.executeCommand(CoreCommandProcessor.java:150)
	at org.jetbrains.kotlin.idea.util.application.ApplicationUtilsKt.executeWriteCommand(ApplicationUtils.kt:38)
	at com.github.valich.jflextokotlin.actions.ConvertLexerToKotlin.convertStringLiterals(ConvertLexerToKotlin.kt:42)
	at com.github.valich.jflextokotlin.actions.ConvertLexerToKotlin.actionPerformed(ConvertLexerToKotlin.kt:27)
	at com.intellij.openapi.actionSystem.ex.ActionUtil.doPerformActionOrShowPopup(ActionUtil.java:315)
	at com.intellij.ide.actions.GotoActionAction.lambda$performActionImpl$4(GotoActionAction.java:100)
	at com.intellij.openapi.actionSystem.ex.ActionUtil.performDumbAwareWithCallbacks(ActionUtil.java:337)
	at com.intellij.ide.actions.GotoActionAction.performActionImpl(GotoActionAction.java:99)
	at com.intellij.ide.actions.GotoActionAction.lambda$performAction$2(GotoActionAction.java:79)
	at com.intellij.openapi.application.TransactionGuardImpl.runWithWritingAllowed(TransactionGuardImpl.java:209)
	at com.intellij.openapi.application.TransactionGuardImpl.access$100(TransactionGuardImpl.java:21)
	at com.intellij.openapi.application.TransactionGuardImpl$1.run(TransactionGuardImpl.java:191)
	at com.intellij.openapi.application.impl.ApplicationImpl.runIntendedWriteActionOnCurrentThread(ApplicationImpl.java:881)
	at com.intellij.openapi.application.impl.ApplicationImpl$3.run(ApplicationImpl.java:513)
	at com.intellij.openapi.application.impl.FlushQueue.doRun(FlushQueue.java:75)
	at com.intellij.openapi.application.impl.FlushQueue.runNextEvent(FlushQueue.java:118)
	at com.intellij.openapi.application.impl.FlushQueue.flushNow(FlushQueue.java:42)
	at java.desktop/java.awt.event.InvocationEvent.dispatch(InvocationEvent.java:318)
	at java.desktop/java.awt.EventQueue.dispatchEventImpl(EventQueue.java:779)
	at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:730)
	at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:724)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:399)
	at java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:86)
	at java.desktop/java.awt.EventQueue.dispatchEvent(EventQueue.java:749)
	at com.intellij.ide.IdeEventQueue.defaultDispatchEvent(IdeEventQueue.java:898)
	at com.intellij.ide.IdeEventQueue._dispatchEvent(IdeEventQueue.java:746)
	at com.intellij.ide.IdeEventQueue.lambda$dispatchEvent$6(IdeEventQueue.java:439)
	at com.intellij.openapi.progress.impl.CoreProgressManager.computePrioritized(CoreProgressManager.java:791)
	at com.intellij.ide.IdeEventQueue.lambda$dispatchEvent$7(IdeEventQueue.java:438)
	at com.intellij.openapi.application.TransactionGuardImpl.performActivity(TransactionGuardImpl.java:105)
	at com.intellij.ide.IdeEventQueue.performActivity(IdeEventQueue.java:604)
	at com.intellij.ide.IdeEventQueue.lambda$dispatchEvent$8(IdeEventQueue.java:436)
	at com.intellij.openapi.application.impl.ApplicationImpl.runIntendedWriteActionOnCurrentThread(ApplicationImpl.java:881)
	at com.intellij.ide.IdeEventQueue.dispatchEvent(IdeEventQueue.java:484)
	at java.desktop/java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:207)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:128)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:117)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:113)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:105)
	at java.desktop/java.awt.EventDispatchThread.run(EventDispatchThread.java:92)
Caused by: java.lang.AssertionError: Unexpected token: '\'
	at com.intellij.lang.java.parser.JavaParserUtil.parseFragment(JavaParserUtil.java:210)
	at com.intellij.psi.impl.source.tree.JavaElementType$JavaDummyElementType.parseContents(JavaElementType.java:273)
	at com.intellij.psi.impl.source.tree.LazyParseableElement.lambda$ensureParsed$2(LazyParseableElement.java:175)
	at com.intellij.psi.impl.DebugUtil.performPsiModification(DebugUtil.java:481)
	at com.intellij.psi.impl.source.tree.LazyParseableElement.ensureParsed(LazyParseableElement.java:174)
	at com.intellij.psi.impl.source.tree.LazyParseableElement.getFirstChildNode(LazyParseableElement.java:226)
	at com.intellij.psi.impl.source.JavaDummyElement.getFirstChildNode(JavaDummyElement.java:69)
	at com.intellij.psi.impl.PsiJavaParserFacadeImpl.createExpressionFromText(PsiJavaParserFacadeImpl.java:293)
	... 83 more
```

**Comment by valich** [comment]
Fixed with #2
